static/js/containers/SignupDialog.js

```diff
@@ -65,44 +37,20 @@ const SignupDialog = ({
         The MIT MicroMasters program is powered by edX.
         To sign up for a MIT MicroMasters program you need an edX account.
       </span>
-      <div className="program-select">
-        Which MicroMasters program are you signing up for?
-        { selectProgram(setProgram, program) }
-      </div>
+
       <a className="mm-button signup" href="/login/edxorg">
         Continue with edX
       </a>
-      <div className="terms-of-service-text">
-        By clicking "Continue with edX" I certify that I agree with
-        <a href="/terms_of_service" target="_blank">
-          {" MIT MicroMasters Terms of Service. "}
-        </a>
-        Read our
-        <a
-          href="http://web.mit.edu/referencepubs/nondiscrimination/index.html" 
-          target="_blank"
-        >
-          { " Nondiscrimination Policy." }
-        </a>
-      </div>
     </div>
   </Dialog>;
 };
 
 const mapStateToProps = state => ({
   open: state.signupDialog.dialogVisibility,
-  program: state.signupDialog.program,
 });
 
-const mapDispatchToProps = dispatch => {
-  const setProgramHelper = (event, key, value) => {
-    dispatch(setProgram(value));
-  };
-
-  return {
-    setDialogVisibility: visible => dispatch(setDialogVisibility(visible)),
-    setProgram: setProgramHelper,
-  };
-};
+const mapDispatchToProps = dispatch => ({
+  setDialogVisibility: visible => dispatch(setDialogVisibility(visible)),
```

Comment:
I think this could be changed to use the `createSimpleActionHelper` function (in `util/redux.js`)